knpu/client-credentials.rst

```diff
@@ -20,13 +19,12 @@ So COOP is real, sort of. You can find this make-believe website by going
 to `http://coop.apps.knpuniversity.com`_. Go ahead and create an account,
 and start controlling your virtual farm. It's the future!
 
-Starting our Command-line Script
---------------------------------
+## Starting our Command-line Script
 
 COOP's API is simple, with just a few endpoints, including the one we
 want for our little command-line script: eggs-collect.
 
-I've already made a ``cron/`` directory with a script called ``collect_eggs.php``
+I've already made a `cron/` directory with a script called `collect_eggs.php`
 that'll get us started::
```

Comment:
Double colon `::` is an .rst feature, so we should replace all `::` at the end with `:` and use \`\`\` for next code block.